src/Nethermind/Nethermind.JsonRpc.Test/Nethermind.JsonRpc.Test.csproj

```diff
@@ -4,6 +4,7 @@
     <IsPackable>false</IsPackable>
     <LangVersion>latest</LangVersion>
     <Nullable>enable</Nullable>
+    <WarningsAsErrors>true</WarningsAsErrors>
```

Comment:
What is the difference between this and TreatWarningsAsErrors?

Reply:
It's a typo. Fixed!